client/config/index.js

```diff
@@ -12,8 +12,4 @@ if ( 'undefined' === typeof window ||  ! window.config ) {
  * @property {Function} anyEnabled whether one of a set of properties is enabled
  * @property {Function} isEnabled whether a specific property is enabled
  */
-const config = window.config;
-config.isEnabled = window.isEnabled;
-config.anyEnabled = window.anyEnabled;
-
-export default config;
+export default require( '../../server/config' );
```

Comment:
I don't think we should ever reference server code inside client. That's why I extracted this code and moved to `client/lib/` in https://github.com/Automattic/wp-calypso/pull/11352/commits/0fd7456166b81e56915f61396b4570b4b79a5983.